test/base.test.js

```diff
@@ -65,16 +98,27 @@ test('should support multiple secrets', async (t) => {
   const fastify = await buildFastify(handler, options, plugin)
   t.teardown(() => fastify.close())
 
-  const response = await fastify.inject({
+  const sessionIdSignedWOldSecret = 'aYb4uTIhdBXCfk_ylik4QN6-u26K0u0e.InCp31AuDa7DX%2F8rGBz8RMFiCpmUtjcF%2BS7Aco7tur8'
```

Comment:
Please dont shorten the variable name. Rename it to sessionIdSignedWithOldSecret.

We are changing a security sensible module. We have as much harddisk space regarding unit tests as possible ;)